### PR TITLE
Improvement of the dotnet-contributing files

### PR DESCRIPTION
**Pull request by Shubhdeep02**
# Description

Improvement of the dotnet-contributing file

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1149

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -123,3 +123,13 @@ A non-exclusive list of code that must be places in `vendor/`:
 ## Code of Conduct
 
 This project has adopted the [Contributor Covenant Code of Conduct](https://github.com/dapr/community/blob/master/CODE-OF-CONDUCT.md)
+
+
+
+## Github Dapr Bot Commands
+
+Checkout the [daprbot documentation](https://docs.dapr.io/contributing/daprbot/) for Github commands you can run in this repo for common tasks. For example, you can run the `/assign` (as a comment on an issue) to assign the issue to yourself.
```
Comment (mikeee) [inline CONTRIBUTING.md:131]:
```suggestion
Checkout the [daprbot documentation](https://docs.dapr.io/contributing/daprbot/) for Github commands you can run in this repo for common tasks. For example, you can comment `/assign` on an issue to assign it to yourself.
```

File: daprdocs/content/en/dotnet-sdk-contributing/dotnet-contributing.md [hunk 2]
```diff
@@ -21,3 +21,7 @@ The `daprdocs` directory contains the markdown files that are rendered into the
 
    - All rules in the [docs guide]({{< ref contributing-docs.md >}}) should be followed in addition to these.
    - All files and directories should be prefixed with `dotnet-` to ensure all file/directory names are globally unique across all Dapr documentation.
+
+## Github Dapr Bot Commands
+
+Checkout the [daprbot documentation](https://docs.dapr.io/contributing/daprbot/) for Github commands you can run in this repo for common tasks. For example, you can run the `/assign` (as a comment on an issue) to assign the issue to yourself.
```
Comment (mikeee) [inline daprdocs/content/en/dotnet-sdk-contributing/dotnet-contributing.md:27]:
```suggestion
Checkout the [daprbot documentation](https://docs.dapr.io/contributing/daprbot/) for Github commands you can run in this repo for common tasks. For example, you can comment `/assign` on an issue to assign it to yourself.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (KrylixZA) [reply CONTRIBUTING.md:131]:
```suggestion
Checkout the [daprbot documentation](https://docs.dapr.io/contributing/daprbot/) for GitHub commands you can run in this repo for common tasks. For example, you can run the `/assign` (as a comment on an issue) to assign the issue to yourself.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -123,3 +123,13 @@ A non-exclusive list of code that must be places in `vendor/`:
 ## Code of Conduct
 
 This project has adopted the [Contributor Covenant Code of Conduct](https://github.com/dapr/community/blob/master/CODE-OF-CONDUCT.md)
+
+
+
+## Github Dapr Bot Commands
```
Comment (KrylixZA) [inline CONTRIBUTING.md:129]:
```suggestion
## GitHub Dapr Bot Commands
```

File: daprdocs/content/en/dotnet-sdk-contributing/dotnet-contributing.md [hunk 4]
```diff
@@ -21,3 +21,7 @@ The `daprdocs` directory contains the markdown files that are rendered into the
 
    - All rules in the [docs guide]({{< ref contributing-docs.md >}}) should be followed in addition to these.
    - All files and directories should be prefixed with `dotnet-` to ensure all file/directory names are globally unique across all Dapr documentation.
+
+## Github Dapr Bot Commands
```
Comment (KrylixZA) [inline daprdocs/content/en/dotnet-sdk-contributing/dotnet-contributing.md:25]:
```suggestion
## GitHub Dapr Bot Commands
```

File: daprdocs/content/en/dotnet-sdk-contributing/dotnet-contributing.md (hunk 2 above)
Comment (KrylixZA) [reply daprdocs/content/en/dotnet-sdk-contributing/dotnet-contributing.md:27]:
```suggestion
Checkout the [daprbot documentation](https://docs.dapr.io/contributing/daprbot/) for GitHub commands you can run in this repo for common tasks. For example, you can run the `/assign` (as a comment on an issue) to assign the issue to yourself.
```

**Comment by WhitWaldo** [comment]
@Shubhdeep02 Would you happen to have any time to review the changes proposed in the next month so we can merge this?

**Review by WhitWaldo — APPROVED** [review]
LGTM

**Comment by WhitWaldo** [comment]
Thank you very much for your contribution @Shubhdeep02 !

**Comment by Shubhdeep02** [comment]
Really appreciate the help @WhitWaldo....If you don't mind can you please suggest me some open source organizations to which I should approach for their internal software role hiring

**Comment by WhitWaldo** [comment]
@Shubhdeep02 I'm sorry - I frankly have no real contacts in the open source world that I could refer you to. I do wish you the best in your job search endeavor and hope you're able to find a role soon!

**Pull request merged**